doc/onboarding.md

```diff
@@ -169,6 +169,7 @@ Landing a PR
     * `Reviewed-By: human <email>`
       * Easiest to use `git log` then do a search
       * (`/Name` + `enter` (+ `n` as much as you need to) in vim)
+      * Only include collaborators who have commented "LGTM"
```

Comment:
Yes, certainly. All of the ctc members are also collaborators.

On Saturday, August 6, 2016, Yorkie Liu <notifications@github.com> wrote:

> In doc/onboarding.md
> <https://github.com/nodejs/node/pull/7183#discussion_r73786541>:
>
> > @@ -169,6 +169,7 @@ Landing a PR
> >      * `Reviewed-By: human <email>`
> >        * Easiest to use `git log` then do a search
> >        * (`/Name` + `enter` (+ `n` as much as you need to) in vim)
> > +      * Only include collaborators who have commented "LGTM"
>
> Does CTC members should be included as well?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/nodejs/node/pull/7183/files/f645b4c8da57891c7cf5b59987548e87e32479fb#r73786541>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAa2eXgC6ANpp1R5Da2oSUQu7NEtHcdmks5qdJUvgaJpZM4Ivauh>
> .
>